aunt-leahs-app/src/constants.jsx

```diff
@@ -104,4 +19,16 @@ export const pages = {
 	VOLUNTEER_SIGN_UP: 'Volunteer Sign Up'
 };
 
-export const requiredText = "Required";
+export const requiredText = 'Required';
+
+export const applicationBaseUrl = 'http://localhost:3000';
+
+export const adminAPIBaseURL = 'https://aunt-leahs-functions-admin.azurewebsites.net';
```

Comment:
Do you think it makes sense to add the /api/ part to the base urls because that is present in every function?

Reply:
Generally, I think you don't because you might use the same url for other stuff, not just api calls (not sure though). But imo forcing the user to add `/api/` actually helps readability and also searchability